HLTrigger/Configuration/python/customizeHLTForPFTrackingPhaseI2017.py

```diff
@@ -0,0 +1,279 @@
+import FWCore.ParameterSet.Config as cms
+
+
+def producers_by_type(process, *types):
+    return (module for module in process._Process__producers.values() if module._TypedParameterizable__type in types)
+    
+def customizeHLTForPFTrackingPhaseI2017(process):
+
+
+	process.ClusterShapeHitFilterESProducer.PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1TkNewFPix.par'
+	process.hltSiPixelDigis.UsePhase1 = cms.bool( True )
+	process.hltSiPixelDigisRegForBTag.UsePhase1 = cms.bool( True )
+	process.hltSiPixelDigisReg.UsePhase1 = cms.bool( True )
```

Comment:
With grep I can see more than one instance of `ClusterShapeHitFilterESProducer` in HLTrigger/Configuration/python/HLT_25ns15e33_v4_cff.py. 

Wouldn't it be better to apply these changes to all instances of `ClusterShapeHitFilterESProducer` (and `SiPixelRawToDigi`) instead of only this/these?